util/dvsim/Testplan.py

```diff
@@ -48,7 +49,7 @@ def __init__(self, raw_dict):
                                f"{self.fields}\n{e}")
 
         # Set the remaining k-v pairs in raw_dict as instance attributes.
-        for k, v in raw_dict:
+        for k, v in raw_dict.items():
```

Comment:
Huh! Were we not running this code before? I would have thought this would have exploded?